docs/usecases/bids-fmriprep-workflow-NP.sh

```diff
@@ -0,0 +1,201 @@
+you don't have it yet - it is free.
+
+EOF
+datalad save -m "DOC: licenses/ directory stub" licenses/
+
+
+#
+# Execution.
+#
+# That is where access to the powerful resource (HPC) etc would be useful.
+# Every of those containerized apps might need custom options to be added.
+#
+#
+
+# Define common parameters for the reproman run
+
+# ReproMan orchestrator to be used - determines how data/results would be
+# transferred and execution protocoled
+# Use  reproman run --list orchestrators  to get an updated list
+RM_ORC=datalad-pair-run  # ,plain,datalad-pair,datalad-local-run
+
+# Which batch processing system supported by ReproMan will be used
+# Use  reproman run --list submitters  to get an updated list
+# RM_SUB=condor,pbs,local
+
+# Which resource to use
+# It would require (if was not done before) to configure
+# a resource where execution will happen.  For now will just use smaug below.
+# TODO: provide pointers to doc ( ;-) )
+# RM_RESOURCE=
+
+#RM_RESOURCE=discovery
+#RM_SUB=PBS
+#
+# Necessary modules to be loaded in that session:
+#  - singularity/2.4.2
+# Necessary installations/upgrades to be done (TODO: contact John)
+#  - datalad (0.11.6, TODO: release first)
+#  - datalad-container
+
+RM_RESOURCE=smaug
+RM_SUB=condor
+
+# TODO: at reproman level allow to specify ORC and SUB for a resource, so there would
+#   be no need to specify for each invocation.  Could be a new (meta) resource such as
+#   "smaug-condor" which would link smaug physical resource with those parameters
+# TODO: point to the issue in ReproMan
+
+# 1. bids-mriqc -- QA
+
+# Q/TODO: Is there a way to execute/reference the container?
+#   for now doing manually
+datalad create -d . -c text2git data/mriqc
+
+: ${RUNNER:=reproman}
+
+unknown_runner () {
+    echo "ERROR: Unknown runner $RUNNER.  Known reproman and datalad" >&2
+    exit 1
+}
+
+# Sample run without any parallelization, and doing both levels (participant and group)
+RUNNER_ARGS=( --input 'data/bids' --output data/mriqc )
+MRIQC_ARGS=( "{inputs}" "{outputs}" participant group )
+case "$RUNNER" in
+    reproman)
+        reproman run --follow -r "${RM_RESOURCE}" --sub "${RM_SUB}" --orc "${RM_ORC}" \
+	    	 --jp container=containers/bids-mriqc "${RUNNER_ARGS[@]}" "${MRIQC_ARGS[@]}";;
+	datalad)
+        datalad containers-run -n containers/bids-mriqc \
+            "${RUNNER_ARGS[@]}" "${MRIQC_ARGS[@]}";;
+    *) unknown_runner;;
+esac
+
+exit 0  # done for now
+
+# ultimately we should be able to parallelize across subjects. Here is the sample invocation for subj 02
+# singularity run containers/images/bids/bids-mriqc--0.15.0.sing  \
+#			data/bids/ data/mriqc/ participant --participant_label 02
+# and at the "group" level should have no --participant_label option
+
+
+reproman run --follow -r "${RM_RESOURCE}" --sub "${RM_SUB}" --orc "${RM_ORC}" \
+  --bp 'thing=thing-*' \
+  --input '{p[thing]}' \
+  sh -c 'cat {p[thing]} {p[thing]} >doubled-{p[thing]}'
```

Comment:
With the latest push to run-subjobs (ac1427736) checked out, try

```
reproman run --follow -r "${RM_RESOURCE}" --sub "${RM_SUB}" --orc "${RM_ORC}" \
  --jp container=containers/bids-mriqc \
  --bp 'pl=02,13' \
  --input data/bids \
  data/bids data/mriqc participant --participant_label '{p[pl]}'
```

I was able to get that [*] to successfully run via condor on smaug.  As you've already experienced, the management of existing datasets is a bit rough, so you may want to use a fresh dataset.

[*] Or more specifically, this script:

<details>
<summary>script</summary>

```sh
#!/bin/sh
set -eu

cd $(mktemp -d --tmpdir=. ds-XXXX)
datalad create -c text2git .
datalad install -d . ///repronim/containers
datalad install -d . -s https://github.com/ReproNim/ds000003-demo data/bids

mkdir licenses/
echo freesurfer.txt > licenses/.gitignore
cat > licenses/README.md <<EOF

Freesurfer
----------

Place your FreeSurfer license into freesurfer.txt file in this directory.
Visit https://surfer.nmr.mgh.harvard.edu/registration.html to obtain one if
you don't have it yet - it is free.

EOF
datalad save -m "DOC: licenses/ directory stub" licenses/

datalad create -d . data/mriqc

reproman run --resource sm --follow \
         --sub condor --orc datalad-pair-run \
         --jp container=containers/bids-mriqc --bp 'pl=02,13' \
         -i data/bids \
         data/bids data/mriqc participant --participant_label '{p[pl]}'
```

</details>